### ISSUE TITLE
Auto détection controller on windows 98

### ISSUE THREAD

**Issue by Massouille78** [schellingb/dosbox-pure#366]
Hello, 

would it be possible to activate the detection of basic controllers such as 2-button, 4-button joystick or gravis gamepad directly in Windows 98?  

when I go to the Windows controller menu under device manager, none is detected even if I activate controller emulation in the Pure dosbox options.

Thanks in advance for your work!

**Comment by Darknior** [comment]
Yes good idea, it can help a lot to have real controllers detected in Windows to play games :)

**Comment by Massouille78** [comment]
![image](https://user-images.githubusercontent.com/64223347/207054661-badb615a-6a9e-459c-9c68-9ac88b0d0f84.jpeg)

See screen of my request

**Comment by cbrunner83** [comment]
*bump*
it would be really great to have this as a feature for Games like X-Wing Alliance that require an installed Joystick.

or if someone could recommend a workaround solution to emulate a Joystick in Windows95/98
that would help out as well. I've only found Tools that emulate Mouse/Keyboard Input with a Joystick so far....

Thanks!
